### ISSUE TITLE
Add Export template for DoNotAirlock Checklist

### ISSUE THREAD

**Issue by hasteur** [IAmPicard/StarTrekTimelinesSpreadsheet#249]
**Provide a Excel Format for exporting to the DoNotAirlock Checklist https://forum.disruptorbeam.com/stt/discussion/127.  Ideally a Excel format export that exports from the current export: name, 2 blank columns and then 2 programatic columns.

Column A should be "Name" 
Column B and C should be blank
Column D should be titled "Active" depends on some programming
Column E should be "Cryo'd" which should match the value of Frozen 

For Column D: If Not Frozen && level = 100 && rarity = max_rarity && equipment.slot * = 1 (i.e. all equipped) Value = 1, else Value = 0

The DNAC helps me decide what crew to freeze off without impacting certain criteria.

**Additional context**
Tool version: **0.8.2-web**
Operating system: **Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36**

**Comment by jgleicher** [comment]
I support this idea, although it will also need a translation table since the names in both tools are different for some characters. I think it's because this tool uses the game names literally, and the DoNotAirlock tool uses human entered names. If you want a straight export/copy/paste, some names will need to be adjusted as part of the process to get the rows to line-up.

"Dark Ages" McCoy being an example, since quote characters sort first so he's always at the top of an export but further down in the DoNotAirlock list as it doesn't include the quotes in the name. 

Even without a translation table though, this feature would be nice.

**Comment by IAmPicard** [comment]
I'm curious how you use the data in the DNA to make decisions? Perhaps I can implement that functionality in the tool itself.

FWIW, while I think Leshy does a great job writing and maintaining the DNA, it serves a slightly different purpose and makes assertions based on that (it's mostly for folks that for one reason or another don't want to use this tool) - for example it assumes everyone is FFFE; it also has some hardcoded assumptions (literally hand-written names of crew for things like Cadets and missions).

**Comment by hasteur** [comment]
I know they serve different purposes, but I use the data from DNA to decide which crew I can "afford" to send to cryostasis and to help me evaluate the legendary of the week against my standing crew when considering all slots

**Comment by jgleicher** [comment]
I don't use the DNA tool as much as I used to, but that's mostly because I'm reaching terminal collection of all but the legendaries. That said, here's what I still sort-of use the DNA tool for:

* One-stop place to see all the characters, to know which I'm missing in an easy to scroll format without too much extra info. I like the color coded aspects of the DNA spreadsheet. 
* Help keep track of which characters I use for cadet missions, voyages, and gauntlet, and therefore don't freeze. 
* * If this tool could formulate the least number of characters to complete every node of all cadet missions, that would be amazing! I know I have a few extra blue characters unfrozen because I don't want to figure out who can cover more nodes. 
* * I also use the tool after events to determine who to re-freeze, that I pulled out for the event.
* Answers whether to keep or freeze the latest rare character, based on highest skill level compared to my other characters.

A year ago I was using the DNA tool as a way to keep crew unfrozen to complete missions and ensure even skill coverage. It seemed to do a good job of providing basic advice of keep/airlock/freeze. I don't mind that it only covers FF/FE crew, although if this tool could track cadet mission crew as well as general crew of any level, that would provide a feature not possible in the DNA tool (Leshy has stated it'd be too complicated given the Google Sheets tool to do partially leveled/equipped crew).

**Comment by jaspotn** [comment]
I use the DNA, the STT 100 spreadsheet as well as the tool as each have
their own purpose.

The DNA i use to see who is listed as freezable due to space and it was
great to keep track of who I needed for mission critical nodes. The
freezers I take as suggestions based on what other crew I have since I put
all my crew in not just FFFE to see how they would compare at full strength.

The tool I use for voyages, gauntlet and item farming. It's also very easy
to know who to dump after a voyage return.

I use the STT 100 sheet to see how the tool compares for crew selection but
again I list all my crew so I know for voyages my 5* will be the majority
even though most aren't FFFE. But it does help when I have a legendary
behold and want to know per voyage which ones are usually in top 12. I also
have added my own sheets for the ships to see which are fully fused so I
can use for replicator fuel. I know the tool shows this but colour coding
my listing gives an instant view.

Lastly I review the big book in link with all above so I can see whom would
help in voyages and gauntlet after a citation.

Tldr; I use every tool and think each creator does an awesome job, each one
has their own unique purpose for me

On Dec 19, 2018 01:11, "jgleicher" <notifications@github.com> wrote:

I don't use the DNA tool as much as I used to, but that's mostly because
I'm reaching terminal collection of all but the legendaries. That said,
here's what I still sort-of use the DNA tool for:

   - One-stop place to see all the characters, to know which I'm missing in
   an easy to scroll format without too much extra info. I like the color
   coded aspects of the DNA spreadsheet.
   - Help keep track of which characters I use for cadet missions, voyages,
   and gauntlet, and therefore don't freeze.
   -
      - If this tool could formulate the least number of characters to
      complete every node of all cadet missions, that would be
amazing! I know I
      have a few extra blue characters unfrozen because I don't want to figure
      out who can cover more nodes.
   -
      - I also use the tool after events to determine who to re-freeze,
      that I pulled out for the event.
   - Answers whether to keep or freeze the latest rare character, based on
   highest skill level compared to my other characters.

A year ago I was using the DNA tool as a way to keep crew unfrozen to
complete missions and ensure even skill coverage. It seemed to do a good
job of providing basic advice of keep/airlock/freeze. I don't mind that it
only covers FF/FE crew, although if this tool could track cadet mission
crew as well as general crew of any level, that would provide a feature not
possible in the DNA tool (Leshy has stated it'd be too complicated given
the Google Sheets tool to do partially leveled/equipped crew).

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/IAmPicard/StarTrekTimelinesSpreadsheet/issues/249#issuecomment-448242979>,
or mute the thread
